src/base/bittorrent/tracker.cpp

```diff
@@ -244,6 +245,9 @@ Http::Response Tracker::processRequest(const Http::Request &request, const Http:
             throw NotFoundHTTPError();
     }
     catch (const HTTPError &error) {
+        LogMsg(tr("Embedded Tracker: HTTP request error: %1 %2")
+                .arg(QString::number(error.statusCode()), error.statusText())
+            , Log::CRITICAL);
```

Comment:
You misuse Log::CRITICAL. It should only be used when qbt cannot continue to operate normally.

Reply:
Ok, I guess I can make everything a `WARNING` or `INFO` instead.

Reply:
Fixed.